### ISSUE TITLE
Only add co-author from GitHub when it's desired

### ISSUE THREAD

**Issue by jonsmithers** [rkotze/git-mob#97]
## Summary

In my experience with my pairs, adding usernames from GitHub (https://github.com/rkotze/git-mob/issues/74) is always accidental.

Just to be super clear, here's an example:

> Let's say I'm Jane Doe, and I'm pairing with Amy Doe and Bob Doe. I go to add them as pairs, expecting to see this:
> 
> ```console
> $ git ad bd
> Jane Doe <jane@example.com>
> Amy Doe <amy@example.com>
> Bob Doe <bob@example.com>
> ```
> 
> But I haven't yet added them to `~/.git-coauthors`, so instead I see:
> 
> ```console
> $ git mob ad bd
> Jane Doe <jane@example.com>
> Daniel Apatin <35623+ad@users.noreply.github.com>
> Ben Dean <146863+bd@users.noreply.github.com>
> ```
> 
> I've inadvertently added GitHub users `ad` and `bd`. And this is persisted in `~/.git-coauthors` file.
 

## Motivation

Avoids confusion/frustration, especially for brand new users.

**Comment by jonsmithers** [comment]
I have a few ideas for solutions:

- Only add users from github if you specify a flag. For example:
  ```console
  $ git mob ad
  ERROR: co-author with initials "ad" not found
  $ git mob --github jonsmithers
  Jane Doe <jane@example.com>
  Jon Smithers <2660313+jonsmithers@users.noreply.github.com>
  ```
- Only add users from github if you confirm a prompt. For example:
  ```console
  $ git mob jonsmithers
  ERROR: Co-author "jonsmithers" not found.
  Would you like to add "Jon Smithers <2660313+jonsmithers@users.noreply.github.com>" from GitHub (y/n)? y

  Jane Doe <jane@example.com>
  Jon Smithers <2660313+jonsmithers@users.noreply.github.com>
   ```
- Make the behavior configurable. For example:
   ```console
   $ git mob ad
   ERROR: co-author with initials "ad" not found.
   To automatically add users from GitHub, run "git config --global mob.github true"
   $ git config --global mob.github true
   $ git mob ad
   Jane Doe <jane@example.com>
   Daniel Apatin <35623+ad@users.noreply.github.com>
   ```

**Comment by rkotze** [comment]
Fixed in version v2.3.3
